pkg/api/types.go

```diff
@@ -1202,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
+	// Fs represents a filesystem-based local volume
+	// +optional
+	Fs *LocalFsVolume
```

Comment:
Did you consider having the "access types" (block or fs) as a separate field instead of this approach? 

```go
type LocalVolumeSource struct {
 NodeName string
 Path string
 AccessType string // possible values are "block" or "fs"
}
```
I see `AccessType` becoming an unique field in Storage Class and PVC as well and so we can have a consistent API